integration-tests/tests/src/hooks/import-app-before.ts

```diff
@@ -32,6 +32,7 @@ export type AppConfigurationRelaxed = {
   timeout?: number;
   multiplexSessions?: boolean;
   baseFilePath?: string;
+  fetchOverride?: any;
```

Comment:
My immediate reaction is that the "override" part of this name is redundant as all of these options override default values.
```suggestion
  fetch?: any;
```